src/media/css/elements--nav.styl

```diff
@@ -23,20 +23,25 @@ $nav--subnav-to-shift = 'mkt-nav-root';
 $nav--transition = 'transform cubic-bezier(0.455, 0.03, 0.515, 0.955) 250ms';
 $nav--to-shift = '#site-header, main';
 $nav--width = 260px;
-$zindex--nav-hamburger-child = 20;
-$zindex--nav-hamburger-root = 30;
 $zindex--main = 50;
 $zindex--nav-normal-child = 60;
 $zindex--header = 70;
+$zindex--page-overlay = 70;
 $zindex--nav-normal-root = 70;
-$zindex--page-overlay = 100;
+$zindex--nav-hamburger = 80;
+$zindex--nav-hamburger-child = 90;
 
 nav--visible() {
-    .mkt-nav--visible & {
+    body.mkt-nav--visible & {
```

Comment:
I will probably move these classes to the wrapper, don't think we need them on the body anymore